lib/jekyll/commands/serve.rb

```diff
@@ -194,6 +194,7 @@ def webrick_opts(opts)
             :JekyllOptions      => opts,
             :DoNotReverseLookup => true,
             :MimeTypes          => mime_types,
+            :MimeTypesJSON      => File.read(File.expand_path("serve/mime_types.json", __dir__)),
```

Comment:
If we decide to follow my suggestion to reduce the scope of this file, then perhaps we can rename this to something like `:MIMETypeCharsets` and call the file something like `mime_type_charsets.json` to match.